.projenrc.js

```diff
@@ -264,6 +264,12 @@ project.release.addJobs({
   },
 });
 
+// replace default service worker script with no-op worker
+const replaceWorker = project.addTask("replace-worker");
+replaceWorker.exec("cp src/no-op-sw.js build/service-worker.js");
+replaceWorker.exec("rm build/service-worker.js.map");
+project.compileTask.spawn(replaceWorker);
```

Comment:
I added the no-op service worker as a JavaScript file directly since the current `service-worker.ts` currently gets treated as special by the `create-react-app` build tools (some plugins try to injecting content into it, and the compiler will yell if I change the file too much). I'd also just rather serve the JavaScript file unminified.